packages/nodes-base/nodes/Slack/V2/MessageDescription.ts

```diff
@@ -557,6 +557,14 @@ export const messageFields: INodeProperties[] = [
 		description: 'Other options to set',
 		placeholder: 'Add options',
 		options: [
+			{
+				displayName: 'Include Link To Workflow',
+				name: 'includeLinkToWorkflow',
+				type: 'boolean',
+				default: true,
+				description:
+					'Whether to append a link to this workflow at the end of the message. This is helpful if you have many workflows sending Slack messages.',
+			},
```

Comment:
this could be hidden by version
```
					displayOptions: {
						hide: {
							"@version": [2],
						},
					},
```

Reply:
The option should still be available for v2 so if people need this functionality for older version of the node they should be able to enable it. But it should be on by default only for version 2.1